.github/ISSUE_TEMPLATE/organization-membership-request.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Organization you are requesting membership in
+e.g. (at)istio
+
+### Company Affiliation or Individual
+e.g. Google, IBM, VMware, Red Hat or XX etc, otherwise, Individual
+
+### Requirements
+- Subscribed to istio-dev@googlegroups.com (istio-dev@googlegroups.com).
```

Comment:
Make this a link:

[istio-dev@](https://groups.google.com/forum/#!forum/istio-dev)